### PR TITLE
Remove dependency version numbers from README.md

### PR DESCRIPTION
**Pull request by ahounsel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ If DNSSEC records are frequently dropped by network middleboxes, then Firefox cl
 ## Libraries
 Our addon utilizes the following Node.js modules:
 
-- browserify [(^16.5.2)](https://github.com/browserify/browserify/tree/v16.5.2)
-- npm-run-all [(^4.1.5)](https://github.com/mysticatea/npm-run-all/tree/v4.1.5)
-- uuid [(^8.3.0)](https://github.com/uuidjs/uuid/tree/v8.3.0)
-- web-ext [(^5.3.0)](https://github.com/mozilla/web-ext/tree/5.3.0)
+- [browserify](https://github.com/browserify/browserify.git)
```
Comment (mythmon) [inline README.md:52]: You probably don't want to link to the `.git` URLs here. Those just redirect back to `https://github.com/browserify/browserify`.

**Review by mythmon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ahounsel) [reply README.md]: I think my new commit addresses this; is this what you had in mind?

**Review by mythmon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mythmon) [reply README.md]: Yep! Looks good now.

**Comment by ahounsel** [comment]
Looks like you approved the changes but the PR didn't merge

**Comment by mythmon** [comment]
Oops, I forgot you don't have rights to merge PRs here. Should we change that?

**Comment by ahounsel** [comment]
Sure, if that makes sense! Presumably I should still get my PRs reviewed by someone in the future, right?

**Comment by mythmon** [comment]
Yeah. I'll set up branch protection so reviews are still required.

**Comment by ahounsel** [comment]
Great, thanks!!

**Pull request merged**